pkg/env/costmodelenv.go

```diff
@@ -84,8 +84,14 @@ const (
 	PrometheusRetryOnRateLimitDefaultWaitEnvVar = "PROMETHEUS_RETRY_ON_RATE_LIMIT_DEFAULT_WAIT"
 
 	IngestPodUIDEnvVar = "INGEST_POD_UID"
+
+	ETLReadOnlyMode = "ETL_READ_ONLY"
 )
 
+func GetETLReadOnlyMode() bool {
```

Comment:
Just a small nit. We've been using `Is` prefixes for boolean values. `IsETLReadOnlyMode()`